### PR TITLE
update plugins documentation

### PR DESCRIPTION
**Pull request by antonk52**
you have to install gem files for plugins for them in order to work

### CODE REVIEW

**Comment by envygeeks** [comment]
I think that would be nicer as point 3, would you mind making it it's own point on the list rather than an addition onto point 2?

**Comment by antonk52** [comment]
perhaps it may make more sense as a sub point to point 2, as it is not a separate way to use a plugin you want to install. To use a plugin you have to install a gem and specify the name in the config. Therefore, it it should be in one point

**Comment by envygeeks** [comment]
Let me rephrase what I said: It's it's own step.

**Comment by antonk52** [comment]
Maybe I did not phrase it write, as jekyll offers you 3 options how to install a plugin, installing a gem will be a part of option 2, isn't it? This is why it should not be a separate point

**Comment by envygeeks** [comment]
I think we are both lost in translation here, I did not fully read that page and it seems we both missed point 3.  Since this applies to both point 2 and point 3 it should be a sub-note outside of all those notes (right outside of it.) I do apologize for not reading the entire page before commenting.

**Comment by envygeeks** [comment]
Let me elaborate, instead of putting it on 2 or 3, put it under all of the points as a ***note*** so that people see it at the bottom since it would be pretty hard not to grep that when reading the page.

**Comment by antonk52** [comment]
It is okay, communication can be difficult at times. I have updated my patch by taking the note outside the list of options.

**Comment by parkr** [comment]
I think this block is confusing in general, because it says they are ways of _installing_ jekyll plugins, though none of them actually do that. They're more a way of enabling them in your site. In point 1, no `gem install` is required, point 2 required `gem install`, and point 3 requires `bundle install`.

**Comment by antonk52** [comment]
I agree on the confusing part, I have edited my patch.

**Comment by parkr** [comment]
Can you run `bundle exec rake site:preview` and fix the source until this looks right?

screen shot 2015-10-08 at 3 33 34 pm

**Comment by antonk52** [comment]
The latest edit is what you meant, right?
